app/components/gh-tags-list-item.js

```diff
@@ -0,0 +1,80 @@
+import $ from 'jquery';
+import Component from '@ember/component';
+import {alias} from '@ember/object/computed';
+import {computed} from '@ember/object';
+import {inject as service} from '@ember/service';
+
+export default Component.extend({
+    ghostPaths: service(),
+    notifications: service(),
+    router: service(),
+
+    tagName: 'li',
+    classNames: ['gh-list-row', 'gh-tags-list-item'],
+    classNameBindings: ['active'],
+
+    active: false,
+
+    // closure actions
+    onClick() {},
+    onDoubleClick() {},
+
+    id: alias('tag.id'),
+    slug: alias('tag.slug'),
+    name: alias('tag.name'),
+    isInternal: alias('tag.isInternal'),
+    description: alias('tag.description'),
+    postsCount: alias('tag.count.posts'),
+    postsLabel: computed('tag.count.posts', function () {
+        let noOfPosts = this.postsCount || 0;
+        return (noOfPosts === 1) ? `${noOfPosts} post` : `${noOfPosts} posts`;
+    }),
+
+    didReceiveAttrs() {
+        if (this.active) {
+            this.scrollIntoView();
+        }
+    },
+
+    click() {
+        this.onClick(this.tag);
+    },
+
+    doubleClick() {
+        this.onDoubleClick(this.tag);
+    },
+
+    scrollIntoView() {
+        let element = this.$();
```

Comment:
We shouldn't be using `this.$()` or `$()` anywhere any more - jQuery is deprecated in Ember and we're trying to move our code away from it so that we can drop the dependency. You can use `this.element` to get the native DOM element of the component.

Reply:
Since we don't need `scrollIntoView` anymore for tags list screen, this is being removed.